src/test/emulators/auth/batch.spec.ts

```diff
@@ -502,3 +503,128 @@ describeAuthEmulator("accounts:batchCreate", ({ authApi }) => {
+      .send({ localIds: [user1.localId, user2.localId] })
+      .then((res) => {
+        expectStatusCode(200, res);
+        expect(res.body.errors ?? []).to.be.empty;
+      });
+  });
+
+  it("should error for accounts not disabled", async () => {
+    const user1 = await registerAnonUser(authApi());
+    const user2 = await registerUser(authApi(), {
+      email: "foobar@example.com",
+      password: "barbaz",
+    });
+    // User 1 not disabled.
+    await updateAccountByLocalId(authApi(), user2.localId, { disableUser: true });
+
+    await authApi()
+      .post(`/identitytoolkit.googleapis.com/v1/projects/${PROJECT_ID}/accounts:batchDelete`)
+      .set("Authorization", "Bearer owner")
+      .send({ localIds: [user1.localId, user2.localId] })
+      .then((res) => {
+        expectStatusCode(200, res);
+
+        expect(res.body.errors).to.eql([
+          {
+            index: 0,
+            localId: user1.localId,
+            message: "NOT_DISABLED : Disable the account before batch deletion.",
+          },
+        ]);
+      });
+  });
+
+  it("should delete disabled and not disabled accounts with force: true", async () => {
+    const user1 = await registerAnonUser(authApi());
+    const user2 = await registerUser(authApi(), {
+      email: "foobar@example.com",
+      password: "barbaz",
+    });
+    // User 1 not disabled.
+    await updateAccountByLocalId(authApi(), user2.localId, { disableUser: true });
+
+    await authApi()
+      .post(`/identitytoolkit.googleapis.com/v1/projects/${PROJECT_ID}/accounts:batchDelete`)
+      .set("Authorization", "Bearer owner")
+      .send({ localIds: [user1.localId, user2.localId], force: true })
+      .then((res) => {
+        expectStatusCode(200, res);
+        expect(res.body.errors ?? []).to.be.empty;
+      });
+  });
+
+  it("should not report errors for nonexistent localIds", async () => {
+    await authApi()
+      .post(`/identitytoolkit.googleapis.com/v1/projects/${PROJECT_ID}/accounts:batchDelete`)
+      .set("Authorization", "Bearer owner")
+      .send({ localIds: ["nosuch", "nosuch2"] })
+      .then((res) => {
+        expectStatusCode(200, res);
+        expect(res.body.errors ?? []).to.be.empty;
+      });
+
+    await authApi()
+      .post(`/identitytoolkit.googleapis.com/v1/projects/${PROJECT_ID}/accounts:batchDelete`)
+      .set("Authorization", "Bearer owner")
+      .send({ localIds: ["nosuch", "nosuch2"], force: true })
+      .then((res) => {
+        expectStatusCode(200, res);
+        expect(res.body.errors ?? []).to.be.empty;
+      });
+  });
+
+  it("should error if localIds array is empty", async () => {
+    await authApi()
+      .post(`/identitytoolkit.googleapis.com/v1/projects/${PROJECT_ID}/accounts:batchDelete`)
+      .set("Authorization", "Bearer owner")
+      .send({ localIds: [], force: true })
+      .then((res) => {
+        expectStatusCode(400, res);
+        expect(res.body.error.message).to.equal("LOCAL_ID_LIST_EXCEEDS_LIMIT");
```

Comment:
Optional: Do we want to make this more explicit? `LOCAL_ID_LIST_EMPTY`? Or is this what we expect from Prod too?

Reply:
Prod behavior, also error codes are parsed by SDKs so we cannot change or introduce freely